tests/openvino/test_exporters_cli.py

```diff
@@ -122,7 +122,7 @@ def test_exporters_cli(self, task: str, model_type: str):
     def test_exporters_cli_tokenizers(self, task: str, model_type: str):
```

Comment:
we shoudl remove :
```diff
- @unittest.skipIf(not is_openvino_tokenizers_available(), reason="OpenVINO Tokenizers not available")
```

Reply:
Removed and add logic checks for ["Test openvino-nightly" stage](https://github.com/huggingface/optimum-intel/blob/a48e0ca3177075235c2a67ee770e07bd751e49dd/.github/workflows/test_openvino.yml#L39), where the libraries are not compatible.